src/core/Message.java

```diff
@@ -359,5 +363,29 @@ public String getAppID() {
 	public void setAppID(String appID) {
 		this.appID = appID;
 	}
+	/**
+	 * @return the msgType
+	 */
+	public int getMsgType(){
```

Comment:
Since this is quite specific to a routing module, I'd suggest using the {add,get,update}Property methods for this kind of functionality.